addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -96,6 +96,29 @@ def _prepare_fatturapa_withholding_values(self):
             })
         return withholdings.values()
 
+    def _prepare_fatturapa_pension_fund_values(self):
+        pension_funds = {}
+        pension_funds_tax_details = self._prepare_edi_tax_details(filter_to_apply=lambda l: l['tax_id'].l10n_it_pension_fund_type)
+        for _tax_name, tax_dict in pension_funds_tax_details['tax_details'].items():
+            base_lines = self.env['account.move.line']
+            for x in tax_dict['group_tax_details']:
+                base_lines += x['base_line_id']
+            taxes = base_lines.mapped("tax_ids")
+            tax = tax_dict['tax']
+            withholding_tax = taxes.filtered(lambda x: x.l10n_it_withholding_type)
```

Comment:
The logic is quite confusing here.  So a cassa can be related to a withholding in itself?  Weird, because normally the withholding is applied on the base amount.